site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -252,13 +260,19 @@ adapter's `HONO_AMQP_AUTHENTICATION_REQUIRED` configuration property to be expli
 
 **Examples**
 
-Publish some JSON data for device `4711`:
+The Hono Sandbox does not support connections from unauthenticated devices. However, if it did, the command for
+publishing some JSON data on behalf of device `4711` of tenant `DEFAULT_TENANT` would look like this:
 
 ~~~sh
-# in directory: hono/cli/target/
-java -jar hono-cli-*-exec.jar --spring.profiles.active=amqp-send --message.address=t/DEFAULT_TENANT/4711
+# in directory: hono/cli-quarkus/target/
+java -jar hono-cli-*-exec.jar amqp --sandbox telemetry --tenant DEFAULT_TENANT --device 4711 --payload '{"foo": "bar"}' --content-type application/json
 ~~~
 
+{{% notice info %}}
+The `--tenant` and `--device` options are required in this case because the AMQP adapter cannot determine the tenant and
+device ID from the authenticated device.
```

Comment:
Sounds a bit strange to me because there is not authenticated device here in the first place.
How about something like:
```
[...] because no authentication data is given here from which the AMQP adapter could obtain the tenant and device ID.
```